demo/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
     entrypoint: sleep
     command: infinity
     volumes:
-      - ./:/conjurinc/ansible/
+      - ./conjur-client-files:/conjurinc/ansible/
```

Comment:
Can we name these folders the same thing, to make the flow easier to understand?